Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -1351,41 +1351,32 @@ public function isMonolithicPhp()
             return false;
         }
 
-        $kinds = $this->findGivenKind(array(T_OPEN_TAG, T_OPEN_TAG_WITH_ECHO, T_INLINE_HTML));
-
-        /*
-         * Fix HHVM incompatibilities
-         */
-        $hhvmOpenTagsWithEcho = array();
-        $hhvmHashBangs = array();
-
-        if (defined('HHVM_VERSION')) {
-            /*
-             * HHVM parses '<?=' as T_ECHO instead of T_OPEN_TAG_WITH_ECHO
-             *
-             * @see https://github.com/facebook/hhvm/issues/4809
-             */
-            $hhvmEchoes = $this->findGivenKind(T_ECHO);
-            foreach ($hhvmEchoes as $token) {
-                if (0 === strpos($token->getContent(), '<?=')) {
-                    $hhvmOpenTagsWithEcho[] = $token;
-                }
+        $openTagsCounter = 0;
+
+        foreach ($this as $token) {
+            if ($token->isGivenKind(T_INLINE_HTML)) {
+                return false;
             }
 
-            /*
-             * HHVM parses "#!/usr/bin/env php\n" as T_HASHBANG (not defined in
```

Comment:
Nice to see this has been removed/dropped (https://3v4l.org/Quf63)
Maybe a good idea to add a test for this, something like:
```php
$this->assertFalse($tokens->isMonolithicPhp("#!/usr/bin/env php\n<?php // comment\n"));
```